var/spack/repos/builtin/packages/libyogrt/package.py

```diff
@@ -52,9 +54,43 @@ def configure_args(self):
 
         sched = self.spec.variants['scheduler'].value
         if sched != "system":
-            args.append('--with-%s=%s' % (sched, self.spec[sched].prefix))
+            if sched == "lsf":
```

Comment:
Can we remove one level of conditional nesting by:
```python
if sched == 'lsf':
    ...
elif sched != 'system':
    ...
```
?

Reply:
Agreed